src/scripts/main.js

```diff
@@ -354,7 +354,24 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((person) => {
+  const row = document.createElement('tr');
 
-// write your code here
+  const data = [
+    person.name,
+    person.sex === 'm' ? 'Male' : 'Female',
+    person.born,
+    person.died,
+    person.died - person.born,
+    Math.ceil(person.died / 100),
+  ];
+
+  data.forEach((value) => {
+    const cell = document.createElement('th');
```

Comment:
Consider using 'td' instead of 'th' for table data cells unless you intend these to be header cells. 'th' is typically used for table headers, not regular data cells.